.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
```

Comment:
## Fix the informing linter errors.
- [ ] Eslint rules are meant to catch bad code and enforce best practices and the eslintrc provided by [Microverse linters repo](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). Please try and fix these, thank you.